charts/charts/haproxy/values.yaml

```diff
@@ -49,8 +49,60 @@ stats:
     username: ''
     password: ''
 
-## open the port for LB and service
-ports: []
```

Comment:
is this removed intentionally?

Reply:
Yes it is no more needed as replaced by defaultAppServers and additionaAppServers parts